services/media_service.ts

```diff
@@ -46,8 +46,12 @@ class MediaService extends SoapService {
           response.writeHead(200, { 'Content-Type': 'image/jpg' });
           response.end(img, 'binary');
         } catch (err) {
-          utils.log.error("Error opening snapshot : %s", err);
-          response.end("404: Not Found: " + request);
+          //utils.log.error("Error opening snapshot : %s", err);
+          //response.end("404: Not Found: " + request);
```

Comment:
I would retain a log entry but changed to a 'debug' statement, this could come in handy later.
Also, lets just delete code instead of commenting it, it can always be found in the git history if needed.